### ISSUE TITLE
Debug can't work

### ISSUE THREAD

**Issue by swq123459** [matepek/vscode-catch2-test-adapter#243]
**Checklist**

- [x] The issue is about this extension and NOT about a fork.
- [x] Check the [known issues](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/support.md#known-issues) list.
- [x] The latest version of the extension was used.
- [x] It is not related to remote-vscode or I checked the following [issue](https://github.com/matepek/vscode-catch2-test-adapter/issues/201)

**Describe the bug**

**To Reproduce**

**Screenshots** (optional)

**Desktop**

- Extension Version:
- VS Code Version: 1.51.1 (x64)
- Catch2 / Google Test / DOCTest Version: 3.6.14
- OS Type and Version: windows10 2004
- Using remote-ssh/docker/wsl?: No

**Regression bug?**

- 
![image](https://user-images.githubusercontent.com/35790719/101564270-cd630c00-3a05-11eb-96ac-ca613d64b1a8.png)
debug can't work
![image](https://user-images.githubusercontent.com/35790719/101564310-df44af00-3a05-11eb-944f-1014f9a36b88.png)

- when i back to version 3.6.9, it work normally

**Comment by emmenlau** [comment]
I also started having this problem in the past few days, maybe two weeks.

The problem comes no matter if I click "Debug" directly above the `TEST` statement in the source code, or I click the Debuggre Symbol in the test explorer UI.

**Desktop**

- Extension Version: 3.6.14
- VS Code Version: 1.51.1
- Catch2 / Google Test / DOCTest Version: Google Test 1.81
- OS Type and Version: Ubuntu 20.04 x86_64
- Using remote-ssh/docker/wsl?: No

**Comment by matepek** [comment]
Logs pls

**Comment by emmenlau** [comment]
The log file generation does not work for me, sadly. Here is my `settings.json`:
```
{
    "testMate.cpp.log.logfile": "/tmp/testMate.cpp.log.logfile.txt",
    "testMate.cpp.debug.breakOnFailure": false,
    "testMate.cpp.test.advancedExecutables": [
        {
            "name": "${filename}",
            "description": "(Test)",
            "pattern": "${os_env:BDASOFTTEMP}/Debug/BDAImageAnalysis/*Test*"
        },
        {
            "name": "${filename}",
            "description": "(Benchmark)",
            "pattern": "${os_env:BDASOFTTEMP}/Release/BDAImageAnalysis/*Benchmark*",
        }
    ]
}
```
After running all tests and closing VSCode, there is no file `/tmp/testMate.cpp.log.logfile.txt` created.

**Comment by emmenlau** [comment]
There is also no relevant output in the `Test Explorer` setting of the OUTPUT tab after running the tests or clicking on `Debug`. Are my settings correct to show a log?

![testmate01](https://user-images.githubusercontent.com/1098087/101652320-657ce780-3a3e-11eb-86ff-3bf00834807e.png)

**Comment by matepek** [comment]
Where the "Text Explorer" text stands there is a dropdown list. If you open it you should found the logs.

**Comment by emmenlau** [comment]
I'm very very sorry but I do not see something helpful there. I can see in the dropdown list the following choices (see screenshot). I clicked on all of them. The most "related" seems to be `Log (Extension Host)`. But this logs only a long list of seemingly irrelevant lines, something with
 - `ExtensionService#loadCommonJSModule`
 - `ExtensionService#_doActivateExtension`
 - `[error] [ms-vscode.cpptools] provider FAILED`
 - `[error] undefined`

Also this list does not change when I click "Debug" in the extension...

![Screenshot from 2020-12-09 17-18-13](https://user-images.githubusercontent.com/1098087/101656660-3e74e480-3a43-11eb-8944-0a16e8b76ca4.png)

**Comment by matepek** [comment]
Oh man, sorry to hear that. I have no idea why you don't have logs. :S
Without that is pretty hard to tell whats went wrong :(
Bellow you see mine.

For workaround you can set up your [own debug config](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/configuration/debug.configTemplate.md)

But I'm still interested to know 
1. why you don't have logs?
2. what caused the current issue.

Try to help me if you can.

<img width="837" alt="Screen Shot 2020-12-09 at 23 25 37" src="https://user-images.githubusercontent.com/16504598/101656990-f28f6300-3a75-11eb-936a-510c585b745f.png">

**Comment by emmenlau** [comment]
Haha yes it seems this is a curious case! But for what its worth, at least my VSCode is not super crazy special: Its a standard Ubuntu installation of the official latest VSCode from Microsoft. I have some 15 or maybe 20 extensions installed, but most of them are rather common ones like "C++ extension", "C# extension" and similar. So I don't think its related to something very "obscure" I'm doing.

Maybe the issue has to do with me using `advancedExecutables`?

We could also try to do a screen share tomorrow, if you have some time?

**Comment by matepek** [comment]
What is your earliest appointment which suites you from now?

**Comment by csorvagep** [comment]
I have the same issue. This feature (configTemplate == null) worked before.
A workaround is to set the configTemplate as explained [here](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/configuration/debug.configTemplate.md), although this is inconvenient.

**Comment by matepek** [comment]
I found an issue which most likely caused this issue. Releasing a fix soon.

**Comment by emmenlau** [comment]
Thanks, let me know when there is something to review!

**Comment by swq123459** [comment]
awe

**Comment by matepek** [comment]
Let me know if the latest version haven't fixed it.

**Comment by swq123459** [comment]
![image](https://user-images.githubusercontent.com/35790719/102101198-47135380-3e65-11eb-98f4-6d64208d7bfe.png)
version 3.16.6 no work for me

**Comment by matepek** [comment]
Fixed in 3.6.17

**Comment by swq123459** [comment]
nice job! It's working
